VideoApp/VideoApp/Stores/TwilioAccessToken/TwilioAccessTokenStore.swift

```diff
@@ -21,27 +21,26 @@ class TwilioAccessTokenStore {
     private let appSettingsStore = AppSettingsStore.shared
     private let authStore = AuthStore.shared
     private let remoteConfigStore = RemoteConfigStoreFactory().makeRemoteConfigStore()
+    
+    func fetchTwilioAccessToken(roomName: String, completion: @escaping (Result<String, APIError>) -> Void) {
```

Comment:
All of the changes in this file is just to restore old code that doesn't use async/await.

It was interesting to try async/await but I need to figure out better strategies for adopting it. The new CallKit code now calls this function.